_config.yml

```diff
@@ -35,15 +35,16 @@ plugins:
 image: "/assets/css/images/logo.png"
 twitter:
   username: techworkersber
+  card: summary
 facebook:
-  app_id: 105285494190738
+  publisher: 105285494190738
```

Comment:
we don't use a facebook app, the publisher key is hopefully correct name for facebook pages